docs/tutorials/infrastructure/binary/index.md

```diff
@@ -0,0 +1,431 @@
+
+```bash
+sudo mv temporal-server /usr/bin/temporal-server
+sudo chmod +x /usr/bin/temporal-server
+```
+
+You'll also need the Temporal UI server. The Temporal UI Server is a standalone binary that can also be obtained from its [Github Releases page](https://github.com/temporalio/ui-server/releases). Download the newest binary for your operating system and extract it on the command line using `curl` and `tar`:
+
+```bash
+curl -OL https://github.com/temporalio/ui-server/releases/download/v2.28.0/ui-server_2.28.0_linux_amd64.tar.gz
+tar -xzf ui-server_2.28.0_linux_amd64.tar.gz
+```
+
+The extracted binary will be called `ui-server`. Move it to a directory on your path like `/usr/bin`:
+
+```bash
+sudo mv ui-server /usr/bin/temporal-ui-server
+sudo chmod +x /usr/bin/temporal-ui-server
+```
+
+At this point, you've downloaded everything you need. The last thing to do is create a `temporal` user on your server that has the appropriate permissions to run the Temporal Service, and directory accessible to this user to store your data in. Run the following commands:
+
+```bash
+sudo useradd temporal
+sudo mkdir /etc/temporal
+sudo chown temporal /etc/temporal
+```
+
+Next, you'll create configuration files for both the Temporal Server and the UI Server in the `/etc` directory.
+
+## Configuring the Temporal Binaries
+
+In this tutorial, you'll configure a connection to a SQLite database, since it doesn't require any additional dependencies. Using your favorite text editor, open a new file called `/etc/temporal-server.yaml`:
+
+```bash
+sudo vim /etc/temporal-server.yaml
+```
+
+Paste the following contents into the file for a starting configuration. You can update any of these values later.
+
+```
+log:
+  stdout: true
+  level: info
+
+persistence:
+  defaultStore: sqlite-default
+  visibilityStore: sqlite-visibility
+  numHistoryShards: 1
+  datastores:
+    sqlite-default:
+      sql:
+        pluginName: "sqlite"
+        databaseName: "/etc/temporal/default.db"
+        connectAddr: "localhost"
+        connectProtocol: "tcp"
+        connectAttributes:
+          cache: "private"
+          setup: true
+
+    sqlite-visibility:
+      sql:
+        pluginName: "sqlite"
+        databaseName: "/etc/temporal/visibility.db"
+        connectAddr: "localhost"
+        connectProtocol: "tcp"
+        connectAttributes:
+          cache: "private"
+          setup: true
+
+global:
+  membership:
+    maxJoinDuration: 30s
+    broadcastAddress: "127.0.0.1"
+  pprof:
+    port: 7936
+
+services:
+  frontend:
+    rpc:
+      grpcPort: 7233
+      membershipPort: 6933
+      bindOnLocalHost: true
+      httpPort: 7243
+
+  matching:
+    rpc:
+      grpcPort: 7235
+      membershipPort: 6935
+      bindOnLocalHost: true
+
+  history:
+    rpc:
+      grpcPort: 7234
+      membershipPort: 6934
+      bindOnLocalHost: true
+
+  worker:
+    rpc:
+      grpcPort: 7239
```

Comment:
`grpcPort` and `bindOnLocalHost` can go for `worker`, it doesn't listen as it doesn't have it's own service. These lines are still present in our defaults/samples just because they haven't been cleaned up yet,